### ISSUE TITLE
Repeats of "RBF onchain txid <txid> (fee <fee>sat) with txid <txid> (fee <fee>sat)"

### ISSUE THREAD

**Issue by tsjk** [ElementsProject/lightning#6479]
### Issue and Steps to Reproduce
* A channel (with a lousy LND-node that couldn't re-negotiate channel closing fees due to a bug they refuse to fix) was force closed
* The funds were eventually released
* The releasing transaction was automatically RBF-d
* After the funds had arrived I get a log entry like twice a minute saying:
`INFO    <node_id_of_node_with_closed_channel>-chan#<id>: RBF onchain txid <txid> (fee <fee>sat) with txid <txid> (fee <fee>sat)`

The txids are the same on both sides of the with, and so are the fees. There was an initial entry of the same kind, where the txids and the fees differed.

Ought I worry about these? And, regardless - I doubt its intended behaviour.

### `getinfo` output
```
{
   "id": "<id>",
   "alias": "<alias>",
   "color": "<color>",
   "num_peers": <less_than_fifty>,
   "num_pending_channels": 0,
   "num_active_channels": <less_than_fifty>,
   "num_inactive_channels": 1,
   "address": [
      {
         "type": "torv3",
         "address": "<onion-address>",
         "port": 9735
      }
   ],
   "binding": [
      {
         "type": "ipv4",
         "address": "127.0.0.1",
         "port": 9735
      }
   ],
   "version": "23.05.2-gentoo-r0",
   "blockheight": 801219,
   "network": "bitcoin",
   "fees_collected_msat": <not_enough_to_get_rich>,
   "lightning-dir": "/var/lib/lightning/bitcoin",
   "our_features": {
      "init": "08a0882a2269a2",
      "node": "88a0882a2269a2",
      "channel": "",
      "invoice": "02000022024100"
   }
}
```

**Comment by grubles** [comment]
I think this should be fixed in master here https://github.com/ElementsProject/lightning/commit/47fd31e8b40f839b4f02c82ac79100a75393c7cc

**Comment by rustyrussell** [comment]
Yes, this spam was fixed in v23.08: we now log duplicates at level DEBUG, and only INFO if things actually change!

**Comment by tsjk** [comment]
I wonder however why these logs appear after the transaction is confirmed (I already got the bitcoin).
